src/vos/tests/vos_tests.c

```diff
@@ -144,7 +148,32 @@ main(int argc, char **argv)
 		return rc;
 	}
 
-	rc = vos_self_init("/mnt/daos");
+	while  ((opt = getopt_long(argc, argv, short_options,
+				   long_options, &index)) != -1) {
+		switch (opt) {
+		case 'S':
+			if (strlen(optarg) >= STORAGE_PATH_LEN) {
+				print_error("%s is longer than STORAGE_PATH_LEN.\n", optarg);
+				goto exit_0;
+			}
+			strncpy(vos_path, optarg, STORAGE_PATH_LEN-1);
+			break;
+		case 'h':
+			print_usage();
+			goto exit_0;
+		default:
+			break;
+		}
+
+		if (vos_path[0] == '/') {
+			break;
+		}
+	}
+	if (vos_path[0] != '/') {
```

Comment:
This might be vos_path[0] != '\0'? could be vos_path something like "./mnt" ? or we only support absolute path?

Reply:
Thanks! I updated the code to use PATH_MAX. Relative path is also allowed now.